demo/src/static/code-markdown/trips.md

```diff
@@ -66,10 +66,10 @@ export default class HeroDemo extends Component {
 trips-layer.js:
 
 ```
-import {Layer, assembleShaders} from 'deck.gl';
 import {Model, Program, Geometry, glGetDebugInfo} from 'luma.gl';
-
-const glslify = require('glslify');
+import {Layer, assembleShaders} from 'deck.gl';
```

Comment:
Nit: Maybe have `deck.gl` before `luma.gl`? Higher level to lower level import order.